test/ILLink.RoslynAnalyzer.Tests/Verifiers/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Testing;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing.Verifiers;
+using Microsoft.CodeAnalysis.Text;
+
+namespace ILLink.RoslynAnalyzer.Tests
+{
+	public partial class CSharpCodeFixVerifier<TAnalyzer, TCodeFix>
```

Comment:
Why do we need this on a separate file and is not included in the  test/ILLink.RoslynAnalyzer.Tests/Verifiers/CSharpCodeFixVerifier`2.cs file?

Reply:
No real reason, it just happened to be the way that the code fix templates laid things out.